types/lingui__react/withI18n.d.ts

```diff
@@ -11,6 +11,7 @@ export interface withI18nOptions {
 
 export interface withI18nProps {
     i18n: I18n;
+    i18nHash: string | null | undefined;
```

Comment:
Was this intended to be optional?  Right now, it has to be provided, but can be null or undefined (which may be intentional).

Reply:
No, it's the `i18nHash` prop is always injected into a component and therefore is a non-optional type:

https://github.com/lingui/js-lingui/blob/7f4cc1546546ff3b0ce63ec4316ce6d0539c670a/packages/react/src/withI18n.js#L65

As @huan086 mentioned: When withHash is true, i18nHash may be null or string. When withHash is false, i18nHash is undefined